src/PepperDash.Essentials.MobileControl.Messengers/Messengers/ISelectableItemsMessenger.cs

```diff
@@ -10,17 +8,37 @@
 namespace PepperDash.Essentials.AppServer.Messengers
 {
     public class ISelectableItemsMessenger<TKey> : MessengerBase
-    {
-        private static readonly JsonSerializer serializer = new JsonSerializer { Converters = { new StringEnumConverter() } };
+    {        
         private readonly ISelectableItems<TKey> itemDevice;
 
         private readonly string _propName;
+
+        /// <summary>
+        /// Constructs a messenger for a device that implements ISelectableItems<typeparamref name="TKey"/>
+        /// </summary>
+        /// <param name="key"></param>
+        /// <param name="messagePath"></param>
+        /// <param name="device"></param>
+        /// <param name="propName"></param>
         public ISelectableItemsMessenger(string key, string messagePath, ISelectableItems<TKey> device, string propName) : base(key, messagePath, device as IKeyName)
         {
             itemDevice = device;
             _propName = propName;
         }
 
+        /// <summary>
+        /// Constructs a messenger for a device that implements IHasInputs<typeparamref name="TKey"/>
```

Comment:
Should this read ISelectableItems instead of IHasInputs?

Reply:
That constructor takes an `IHasInputs<T>`,. But it is being removed in favor of using the `IHasInputsMessenger`